xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/XWikiServletResponse.java

```diff
@@ -66,9 +68,21 @@ public void sendRedirect(String redirect) throws IOException
             LOGGER.warn("Possible HTTP Response Splitting attack, attempting to redirect to [{}]", redirect);
             return;
         }
+        if (redirect.matches("[a-z0-9]+://.*")) {
+            // Full URL, check that the hostname matches
+            if (!getURLSecurityManager().isDomainTrusted(new URL(redirect))) {
+                LOGGER.warn("Possible phishing attack, attempting to redirect to [{}]", redirect);
```

Comment:
I think the message should be improved and we should mention it's not in the list of trusted domains and explain ho to add to it.

Reply:
Also the message should say what is done, i.e. that the redirect has not been performed.

Reply:
> we should mention it's not in the list of trusted domains

+1